content/en/logs/log_collection/javascript.md

```diff
@@ -55,75 +79,78 @@ In order to not miss any logs or errors, you should load and configure the libra
+      });
+    </script>
+  </head>
+</html>
 ```
 
 {{% /tab %}}
 {{< /tabs >}}
 
 **Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
 
+### Initialization parameters
+
+The following parameters can be used to configure the Datadog browser log library to send logs to Datadog:
+
+| Parameter             | Type    | Required | Default | Description                                                                                              |
+|-----------------------|---------|----------|---------|----------------------------------------------------------------------------------------------------------|
+| `clientToken`         | String  | Yes      | `-`     | A [Datadog Client Token][4].                                                                             |
+| `datacenter`          | String  | Yes      | `us`    | The Datadog Site of your organization. `us` for Datadog US site, `eu` for Datadog EU site.               |
+| `forwardErrorsToLogs` | Boolean | no       | `true`  | Set to `false` to stop forwarding console.error logs, uncaught exceptions and network errors to Datadog. |
+| `sampleRate`          | Number  | no       | `100`   | Percentage of sessions to track. Only tracked sessions send logs. `100` for all, `0` for none of them.   |
+
 ## Send a custom log entry
 
-Send a custom log entry directly to Datadog with the `log` function:
+Once Datadog Browser log library is initialized, send a custom log entry directly to Datadog with the API:
 
-```text
-window.DD_LOGS && DD_LOGS.logger.log(<MESSAGE>,<JSON_ATTRIBUTES>,<STATUS>)
-```
+`logger.debug | info | warn | error (message: string, messageContext = Context)`
 
-| Placeholder         | Description                                                                             |
-|---------------------|-----------------------------------------------------------------------------------------|
-| `<MESSAGE>`         | The message of your log that is fully indexed by Datadog                                |
-| `<JSON_ATTRIBUTES>` | A valid JSON object that includes all attributes attached to the `<MESSAGE>`            |
-| `<STATUS>`          | Status of your log; the accepted status values are `debug`, `info`, `warn`, or `error`. |
+{{< tabs >}}
+{{% tab "NPM" %}}
 
-Status can also be used as a placeholder for the `log` function `DD_LOGS.logger.debug(<MESSAGE>,<JSON_ATTRIBUTES>)`.
+```javascript
+import { datadogLogs } from '@datadog/browser-logs';
 
-**Example:**
+datadogLogs.logger.info('Button clicked', { name: 'buttonName', id: 123 });
+```
 
-```js
-...
-<script>
-...
+{{% /tab %}}
+{{% tab "Bundle" %}}
+
+```javascript
 window.DD_LOGS && DD_LOGS.logger.info('Button clicked', { name: 'buttonName', id: 123 });
-...
-</script>
-...
 ```
 
+**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
+
+{{% /tab %}}
+{{< /tabs >}}
+
 This gives the following result:
 
 ```json
```

Comment:
@hdelaby I believe that now it adds view.url and not http.url by default, is that correct?

Reply:
Yes indeed
![Image 2020-01-08 at 1 18 15 PM](https://user-images.githubusercontent.com/21019740/71977615-8975ee80-3219-11ea-9b28-676e70ecc45d.png)

Reply:
fixed !